### DISCUSSION TITLE
Any way to speed up Perplexica?

### DISCUSSION THREAD

**Discussion by tqdo** [ItzCrazyKns/Vane#374] [General]
First, thanks a lot for making this amazing tool. I have been using it for a while and absolutely love it. I know that all [components](https://github.com/ItzCrazyKns/Perplexica/blob/master/docs/architecture/WORKING.md) of Perplexica can be open-source. However, I wonder whether there is any way I can speed up the response time by using some paid service? For example, I wonder instead of using Searxng (I am not familiar with this tool), if we use some paid SERP API, will the result be returned faster? Thanks again.

**Comment by ItzCrazyKns** [comment]
Hi, I am working on adding support for Serp API and on adding multiple performance modes such as quality, performance and balanced that will enhance the user experience. Should be available in the next release. At least some part of it

**Comment by tqdo** [comment]
Thanks a lot. Besides those, do you know any other way I can personally speed up Perplexica, perhaps by paying for a paid service myself?
